### ISSUE TITLE
Unittests in my project complain about javax.validation missing from the classpath

### ISSUE THREAD

**Issue by dnouls** [ManfredTremmel/gwt-bean-validators#5]
When I compile my GWTTestCases I get this WARNING:
```
125  [main] INFO  org.eclipse.jetty.server.Server  - jetty-9.2.14.v20151106
484  [main] WARN  org.eclipse.jetty.annotations.AnnotationConfiguration  - ServletContainerInitializers: detected. Class hierarchy: empty
983  [main] INFO  org.eclipse.jetty.server.handler.ContextHandler  - Started c.g.g.j.@26abb146{/,file:/C:/Development/git-repos/acme/platform/main/multitenant/event-gwt/target/gwt-tests/www/,AVAILABLE}{C:\Development\git-repos\acme\platform\main\multitenant\event-gwt\target\gwt-tests\www}
1030 [main] INFO  org.eclipse.jetty.server.ServerConnector  - Started ServerConnector@50313382{HTTP/1.1}{0.0.0.0:5230}
1030 [main] INFO  org.eclipse.jetty.server.Server  - Started @2921ms
Compiling module com.acme.cloud.platform.multitenant.event.SerializationTest.JUnit
   Computing all possible rebind results for 'com.google.gwt.logging.client.LogConfiguration'
      Rebinding com.google.gwt.logging.client.LogConfiguration
         Checking rule <generate-with class='com.google.gwt.validation.rebind.ValidatorGenerator'/>
            [WARN] Detected warnings related to 'javax.validation.Validator'.   Are validation-api-<version>.jar and validation-api-<version>-sources.jar on the classpath?
            [WARN] Unknown type 'javax.validation.Validator' specified in deferred binding rule
```

The classpath however contains both java.validation 1.1 and the source jar:

```
[DEBUG] boot(compact) classpath:  surefire-booter-2.20.jar  surefire-api-2.20.jar  surefire-logger-api-2.20.jar  test-classes  classes  gwt-user-2.8.1.jar  jsinterop-annotations-1.0.1.jar  jsinterop-annotations-1.0.1-sources.jar  javax.servlet-api-3.1.0.jar  sac-1.3.jar  common.query-gwt-1.0.6-SNAPSHOT.jar  common.query-1.0.6-SNAPSHOT.jar  jackson-databind-2.8.8.jar  jackson-core-2.8.8.jar  gwt-dev-2.8.1.jar  gson-2.6.2.jar  asm-5.0.3.jar  asm-util-5.0.3.jar  asm-tree-5.0.3.jar  asm-commons-5.0.3.jar  colt-1.2.0.jar  ant-1.6.5.jar  commons-collections-3.2.2.jar  commons-io-2.4.jar  icu4j-50.1.1.jar  tapestry-4.0.2.jar  htmlunit-2.19.jar  xalan-2.7.2.jar  serializer-2.7.2.jar  httpclient-4.5.1.jar  httpcore-4.4.3.jar  httpmime-4.5.1.jar  commons-codec-1.10.jar  htmlunit-core-js-2.17.jar  xercesImpl-2.11.0.jar  xml-apis-1.4.01.jar  nekohtml-1.9.22.jar  cssparser-0.9.18.jar  websocket-client-9.2.13.v20150730.jar  websocket-common-9.2.14.v20151106.jar  websocket-api-9.2.14.v20151106.jar  jetty-webapp-9.2.14.v20151106.jar  jetty-xml-9.2.14.v20151106.jar  jetty-servlet-9.2.14.v20151106.jar  jetty-security-9.2.14.v20151106.jar  jetty-servlets-9.2.14.v20151106.jar  jetty-continuation-9.2.14.v20151106.jar  jetty-http-9.2.14.v20151106.jar  jetty-util-9.2.14.v20151106.jar  jetty-io-9.2.14.v20151106.jar  jetty-annotations-9.2.14.v20151106.jar  jetty-plus-9.2.14.v20151106.jar  jetty-jndi-9.2.14.v20151106.jar  javax.annotation-api-1.2.jar  apache-jsp-9.2.14.v20151106.jar  jetty-server-9.2.14.v20151106.jar  jetty-schemas-3.1.M0.jar  apache-jsp-8.0.9.M3.jar  apache-el-8.0.9.M3.jar  gwt-jackson-guava-0.14.1.jar  cloud.platform.multitenant.event-0.0.1-SNAPSHOT.jar  cloud.platform.topology.common-0.0.1-SNAPSHOT.jar  common.crypto-1.0.6-SNAPSHOT.jar  jackson-dataformat-smile-2.8.8.jar  annotations-3.0.1.jar  jcip-annotations-1.0.jar  kafka-clients-0.10.1.0.jar  lz4-1.3.0.jar  snappy-java-1.1.2.6.jar  bcprov-jdk15on-1.54.jar  bcpkix-jdk15on-1.54.jar  jersey-container-grizzly2-http-2.25.1.jar  javax.inject-2.5.0-b32.jar  grizzly-http-server-2.3.28.jar  grizzly-http-2.3.28.jar  grizzly-framework-2.3.28.jar  jersey-common-2.25.1.jar  jersey-guava-2.25.1.jar  hk2-api-2.5.0-b32.jar  hk2-utils-2.5.0-b32.jar  aopalliance-repackaged-2.5.0-b32.jar  hk2-locator-2.5.0-b32.jar  javassist-3.20.0-GA.jar  osgi-resource-locator-1.0.1.jar  jersey-server-2.25.1.jar  jersey-client-2.25.1.jar  jersey-media-jaxb-2.25.1.jar  javax.ws.rs-api-2.0.1.jar  jersey-media-json-jackson-2.25.1.jar  jersey-entity-filtering-2.25.1.jar  jackson-jaxrs-base-2.8.8.jar  jackson-jaxrs-json-provider-2.8.8.jar  jackson-module-jaxb-annotations-2.8.8.jar  kafka_2.11-0.10.1.0.jar  jopt-simple-4.9.jar  metrics-core-2.2.0.jar  scala-library-2.11.8.jar  zkclient-0.9.jar  zookeeper-3.4.8.jar  jline-0.9.94.jar  netty-3.7.0.Final.jar  scala-parser-combinators_2.11-1.0.4.jar  hibernate-validator-5.4.1.Final.jar  javax.el-3.0.1-b08.jar  cloud.platform.app.event-0.0.1-SNAPSHOT.jar  cloud.platform.topology.secure.event-0.0.1-SNAPSHOT.jar  cloud.platform.topology.common-gwt-0.0.1-SNAPSHOT.jar  cloud.platform.topology.common-0.0.1-SNAPSHOT-sources.jar  cloud.platform.app.event-gwt-0.0.1-SNAPSHOT.jar  cloud.platform.topology.secure.event-gwt-0.0.1-SNAPSHOT.jar  gwt-jackson-0.14.1.jar  jackson-annotations-2.8.8.jar  jackson-annotations-2.8.4-sources.jar  javapoet-1.8.0.jar  guava-gwt-21.0.jar  jsr305-3.0.1.jar  error_prone_annotations-2.0.15.jar  j2objc-annotations-1.1.jar  common.config-gwt-1.0.6-SNAPSHOT.jar  common.config-1.0.6-SNAPSHOT.jar  common.io-1.0.6-SNAPSHOT.jar  common.injection-1.0.6-SNAPSHOT.jar  common.core-gwt-1.0.6-SNAPSHOT.jar  common.core-1.0.6-SNAPSHOT.jar  gin-2.1.2.jar  validation-api-1.1.0.Final-sources.jar  validation-api-1.1.0.Final.jar  gwt-bean-validators-0.43.1.jar  hibernate-validator-5.4.1.Final-sources.jar  jboss-logging-3.3.0.Final.jar  classmate-1.3.1.jar  mt-bean-validators-0.43.1.jar  log4j-1.2.17.jar  slf4j-api-1.7.25.jar  slf4j-log4j12-1.7.25.jar  javax.el-api-2.2.5.jar  javax.el-2.2.6.jar  commons-lang3-3.4.jar  commons-validator-1.6.jar  gwt-commons-lang3-3.6-0.jar  gwt-commons-validator-1.6-0.jar  commons-beanutils-1.9.2.jar  commons-digester-1.8.1.jar  commons-logging-1.2.jar  guava-21.0.jar  guice-4.0.jar  javax.inject-1.jar  aopalliance-1.0.jar  guice-assistedinject-4.0.jar  guice-multibindings-4.0.jar  guice-throwingproviders-4.0.jar  mockito-core-2.8.9.jar  byte-buddy-1.6.14.jar  byte-buddy-agent-1.6.14.jar  objenesis-2.5.jar  junit-4.12.jar  hamcrest-core-1.3.jar  common.testing-1.0.6-SNAPSHOT.jar  java  java  surefire-junit4-2.20.jar
```

What am I missing ?

**Comment by ManfredTremmel** [comment]
The generator com.google.gwt.validation.rebind.ValidatorGenerator is replaced by de.knightsoftnet.validators.rebind.ValidatorGenerator in gwt-bean-validators and shouldn't be required anymore. The google version only works with old validation api, not with 1.1, therefor it warns because of missing libs.
I can't change the test inside gwt, so I can't switch of the warning.
Do you have any problem with the functionality, or is it only the warning?

**Comment by dnouls** [comment]
Sorry, I made a mistake. I did not put a inherits in the test gwt.xml file. Once I did that the warning went away.
